google-cloud-bigtable/lib/google/cloud/bigtable/table_admin_client.rb

```diff
@@ -227,6 +227,280 @@ def drop_row_range \
           )
         end
 
+        # This is a private alpha release of Cloud Bigtable snapshots. This feature
+        # is not currently available to most Cloud Bigtable customers. This feature
+        # might be changed in backward-incompatible ways and is not recommended for
+        # production use. It is not subject to any SLA or deprecation policy.
+        #
+        # Creates a new snapshot in the specified cluster from the specified
+        # source table. The cluster and the table must be in the same instance.
+        #
+        # @param table_id [String]
+        #   The unique name of the table to have the snapshot taken.
+        # @param cluster_id [String]
+        #   The name of the cluster where the snapshot will be created in.
+        # @param snapshot_id [String]
+        #   The ID by which the new snapshot should be referred to within the parent
+        #   cluster, e.g., +mysnapshot+ of the form: +[_a-zA-Z0-9][-_.a-zA-Z0-9]*+
+        # @param description [String]
+        #   Description of the snapshot.
+        # @param ttl [Integer]
+        #   The amount of time in seconds that the new snapshot can stay active
+        #   after it is created. Once 'ttl' expires, the snapshot will get
+        #   deleted. The maximum amount of time a snapshot can stay active is
+        #   7 days. If 'ttl' is not specified, the default value of 24 hours
+        #   will be used.
+        # @param options [Google::Gax::CallOptions]
```

Comment:
We don't usually expose `Google::Gax::CallOptions` in the high level API.

Reply:
Got it. I will update in all other APIs also. 

In some of the APIs call options require i.e In `drop_rows` BigTable server is responding with Error - increase timeout limit and try again.
I am planning to add an option for `timeout` for such particular API call.